src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java

```diff
@@ -1841,65 +1810,12 @@ else if (clickedBlock != null &&
-            spawn_eggs.add(Material.DOLPHIN_SPAWN_EGG);
-            spawn_eggs.add(Material.DONKEY_SPAWN_EGG);
-            spawn_eggs.add(Material.DROWNED_SPAWN_EGG);
-            spawn_eggs.add(Material.ELDER_GUARDIAN_SPAWN_EGG);
-            spawn_eggs.add(Material.ENDERMAN_SPAWN_EGG);
-            spawn_eggs.add(Material.ENDERMITE_SPAWN_EGG);
-            spawn_eggs.add(Material.EVOKER_SPAWN_EGG);
-            spawn_eggs.add(Material.GHAST_SPAWN_EGG);
-            spawn_eggs.add(Material.GUARDIAN_SPAWN_EGG);
-            spawn_eggs.add(Material.HOGLIN_SPAWN_EGG);
-            spawn_eggs.add(Material.HORSE_SPAWN_EGG);
-            spawn_eggs.add(Material.HUSK_SPAWN_EGG);
-            spawn_eggs.add(Material.LLAMA_SPAWN_EGG);
-            spawn_eggs.add(Material.MAGMA_CUBE_SPAWN_EGG);
-            spawn_eggs.add(Material.MOOSHROOM_SPAWN_EGG);
-            spawn_eggs.add(Material.MULE_SPAWN_EGG);
-            spawn_eggs.add(Material.OCELOT_SPAWN_EGG);
-            spawn_eggs.add(Material.PARROT_SPAWN_EGG);
-            spawn_eggs.add(Material.PHANTOM_SPAWN_EGG);
-            spawn_eggs.add(Material.PIG_SPAWN_EGG);
-            spawn_eggs.add(Material.PIGLIN_SPAWN_EGG);
-            spawn_eggs.add(Material.POLAR_BEAR_SPAWN_EGG);
-            spawn_eggs.add(Material.PUFFERFISH_SPAWN_EGG);
-            spawn_eggs.add(Material.RABBIT_SPAWN_EGG);
-            spawn_eggs.add(Material.SALMON_SPAWN_EGG);
-            spawn_eggs.add(Material.SHEEP_SPAWN_EGG);
-            spawn_eggs.add(Material.SHULKER_SPAWN_EGG);
-            spawn_eggs.add(Material.SILVERFISH_SPAWN_EGG);
-            spawn_eggs.add(Material.SKELETON_SPAWN_EGG);
-            spawn_eggs.add(Material.SKELETON_HORSE_SPAWN_EGG);
-            spawn_eggs.add(Material.SLIME_SPAWN_EGG);
-            spawn_eggs.add(Material.SPIDER_SPAWN_EGG);
-            spawn_eggs.add(Material.SQUID_SPAWN_EGG);
-            spawn_eggs.add(Material.STRAY_SPAWN_EGG);
-            spawn_eggs.add(Material.STRIDER_SPAWN_EGG);
-            spawn_eggs.add(Material.TROPICAL_FISH_SPAWN_EGG);
-            spawn_eggs.add(Material.TURTLE_SPAWN_EGG);
-            spawn_eggs.add(Material.VEX_SPAWN_EGG);
-            spawn_eggs.add(Material.VILLAGER_SPAWN_EGG);
-            spawn_eggs.add(Material.VINDICATOR_SPAWN_EGG);
-            spawn_eggs.add(Material.WITCH_SPAWN_EGG);
-            spawn_eggs.add(Material.WITHER_SKELETON_SPAWN_EGG);
-            spawn_eggs.add(Material.WOLF_SPAWN_EGG);
-            spawn_eggs.add(Material.ZOGLIN_SPAWN_EGG);
-            spawn_eggs.add(Material.ZOMBIE_SPAWN_EGG);
-            spawn_eggs.add(Material.ZOMBIE_HORSE_SPAWN_EGG);
-            spawn_eggs.add(Material.ZOMBIE_VILLAGER_SPAWN_EGG);
-            spawn_eggs.add(Material.ZOMBIFIED_PIGLIN_SPAWN_EGG);
-
             for (Material material : Material.values())
             {
-                if (!material.isLegacy() && material.name().endsWith("_DYE"))
+                if (material.isLegacy()) continue;
```

Comment:
This check is actually unnecessary - since GP is a non-legacy plugin, legacy materials should not be being provided. I left it in because I don't want to remove it without discussing, but it should be considered.

Reply:
Yea, I'm trying to recall when/why I added this, or if this came from a PR...